README.md

```diff
@@ -152,15 +151,6 @@ OCamlFormat is documented in its man page and through its internal help:
 
 You can also view it [online](ocamlformat-help.txt).
 
-## Reason
-
-OCamlFormat is influenced by and follows the same basic design as `refmt` for [Reason](https://github.com/facebook/reason), but outputs OCaml instead of Reason.
-
-Support for converting Reason to OCaml is included in the `ocamlformat_reason` package, which works by using `refmt` to parse Reason into an OCaml parse tree. The Reason converter can be installed using `opam`:
-```
-opam pin add ocamlformat_reason https://github.com/ocaml-ppx/ocamlformat.git
-```
-
```

Comment:
Maybe we should keep a note there to tell users that we do not interact with Reason code, but redirect them to `refmt`